src/protagonist/API/Features/Storage/Converters/ImageStorageConverter.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace API.Features.Storage.Converters;
+
+/// <summary>
+/// Conversion between API and EF forms of ImageStorage resource
+/// </summary>
+public static class ImageStorageConverter
+{
+    /// <summary>
+    /// Convert ImageStorage entity to API resource
+    /// </summary>
+    public static DLCS.HydraModel.ImageStorage ToHydra(this DLCS.Model.Assets.ImageStorage imageStorage, string baseUrl)
+    {
+        var hydraImageStorage = new DLCS.HydraModel.ImageStorage(baseUrl, imageStorage.Customer,
+            imageStorage.Space, imageStorage.Id.Asset)
+        {
+            ThumbnailSize = (int)imageStorage.ThumbnailSize,
+            Size = (int)imageStorage.Size,
```

Comment:
Rather than cast these to `int`, make the hydramodel properties `long`. This could be the total number of bytes for all stored data for a customer so could be a large value.

Reply:
Nice catch - will do.